.github/workflows/Tagged.yaml

```diff
@@ -0,0 +1,157 @@
+name: Tagged build
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+    paths-ignore:
+      - '.github/workflows/Untagged.yaml'
+  workflow_dispatch:
+
+env:
+  Configuration: Release
+  DSOALRepo: kcat/dsoal
+  DSOALBranch: ${{github.ref_name}}
+  OpenALSoftRepo: kcat/openal-soft
+  OpenALSoftBranch: master
+
+permissions:
+  contents: write
```

Comment:
What is this for, out of curiosity? I don't recall needing anything like this for putting together OpenAL Soft pre-release packages, and I'm not really sure what it does.

Reply:
I was getting a [`Resource not accessible by integration`](https://stackoverflow.com/questions/70435286/resource-not-accessible-by-integration-on-github-post-repos-owner-repo-ac) error in the Github release step [without it](https://github.com/BinauralAudio/dsoal/actions/runs/10327767083). and [adding it seemed to fix it](https://github.com/BinauralAudio/dsoal/actions/runs/10327818703).
But weirdly enough, I tested it again on this account and [it worked just fine](https://github.com/ThreeDeeJay/dsoal/actions?query=branch%3Apermission) without it🤔 
Maybe it was some quirk when using the 2nd fork on my test account, but I guess I'll just remove it in the PR/master branch and hope it doesn't break again. 🤞

Reply:
Wait, what the...
Now I got the error *before* I removed `permissions: contents: write`
https://github.com/ThreeDeeJay/dsoal/actions?query=branch%3Amaster+event%3Apush
So I guess the error was caused by something entirely different. Maybe the last run blocked the previous run from being able to modify releases or something 🤔